src/controllers/auth.controllers.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable arrow-parens */
```

Comment:
some tests are failing

Reply:
I should change some stuff in tests but havent done yet